implementations/rust/ockam/ockam_api/src/cloud/share/create.rs

```diff
@@ -0,0 +1,163 @@
+use minicbor::{Decode, Encode};
+use serde::Serialize;
+
+use super::*;
+
+#[derive(Clone, Debug, Decode, Encode, Serialize)]
+#[cbor(map)]
+#[rustfmt::skip]
+pub struct CreateInvite {
+    #[n(1)] pub expires_at: Option<String>,
```

Comment:
Can we use a `Timestamp` datatype instead? There is one used for credential data (see `credential_data.rs`) that we could reuse.

Reply:
Currently the backend expects returns string-encoded ISO8601 timestamps, can `Timestamp` de/serialize to and from that? At a glance it looks like its inner representation is Unix epoch time instead.

Reply:
I captured this desire as well as part of #5517.